### ISSUE TITLE
setup.py is broken

### ISSUE THREAD

**Issue by basnijholt** [Microsoft/qmt#24]
`pip install .` should install all Python dependencies, which doesn't happen ATM.

Adding a `requirements.txt` file (as is standard) like:
```
numpy
scipy
matplotlib
pytest
future
pytablewriter
```

and doing this inside `setup.py`
```python
with open('requirements.txt') as f:
    requirements = [x.strip() for x in f.readlines()]

setup(
    ...
    install_requires=requirements,
)
```

would fix this issue.

**Comment by johnkgamble** [comment]
I don't want to have both a requirements.txt and an environment.yml file. Thoughts?

**Comment by basnijholt** [comment]
the `emvironment.yml` file can just be:
```
name: qmt
dependencies:
  - python=2.7
  - pyside
```

**Comment by johnkgamble** [comment]
But then you'd need to run pip inside of your new conda environment, right?

**Comment by basnijholt** [comment]
pyside is the only package that has non-Python dependencies, so it needs to be installed somehow.
`conda install pyside; pip install .` should be enough to install this package.

**Comment by johnkgamble** [comment]
OK, cool - please feel free to change this. I'll accept the PR as long as travis doesn't break.

**Comment by basnijholt** [comment]
Closed by #25.
